crates/wasmtime/src/config.rs

```diff
@@ -1710,7 +1721,14 @@ pub enum Strategy {
 
     /// Currently the default backend, Cranelift aims to be a reasonably fast
     /// code generator which generates high quality machine code.
+    #[cfg(feature = "cranelift")]
+    #[cfg_attr(nightlydoc, doc(cfg(feature = "cranelift")))]
     Cranelift,
+
+    /// TODO
+    #[cfg(feature = "winch")]
+    #[cfg_attr(nightlydoc, doc(cfg(feature = "winch")))]
+    Winch,
```

Comment:
I think the `Strategy` variants for Cranelift and Winch need to be present even if the corresponding features are disabled. Both `cli-flags` and the `build_compiler` function here will fail to build if either is disabled otherwise.

Reply:
Good catch here! I've removed the feature gates, so we can let the errors throw when building the compiler if the right features aren't set.